README.md

```diff
@@ -37,10 +33,21 @@ The latest version of Operator Controller can be installed with the following co
 curl -L -s https://github.com/operator-framework/operator-controller/releases/latest/download/install.sh | bash -s
 ```
 
-### Create a ClusterCatalog
+## Getting Started with OLM v1
+
+The following steps will guide you through the process of catalog deployment, and extension installation, upgrade (or downgrade), and uninstallation.
```

Comment:
Nit: I find that turning sentences with a few embedded clauses into a list makes it a little easier to scan.
```suggestion
The following steps will guide you through the following processes:

 * Deploying a catalog
 * Installing, upgrading, or downgrading an extension
 * Deleting catalogs and extensions
```